core/src/databases/table_schema.rs

```diff
@@ -188,13 +188,23 @@ impl TableSchema {
 
                 if let Some(column) = schema.iter_mut().find(|c| &c.name == k) {
                     if column.value_type != value_type {
-                        return Err(anyhow!(
-                            "Field {} has conflicting types on row {}: {:?} and {:?}",
-                            k,
-                            row_index,
-                            column.value_type,
-                            value_type
-                        ));
+                        use TableSchemaFieldType::*;
```

Comment:
is this normal in Rust to have imports elsewhere than on top of the file? noob question here :child:

Reply:
This is not an import (`TableSchemaFieldType` is defined in this file), but it allows to not have to type `TableSchemaFieldType::Float` etc each time. I prefer to have it close to where I'm actually using because:
- I don't want to pollute the whole file's namespace with these very generic sounding symbols
- I think if I were to use eg `Int` elsewhere in the file it wouldn't be very readable ("where is this thing coming from ?>")

no idea if it's idiomatic or not in rust to do like this. If it isn't it should :p